examples/nextjs-notification-settings/app/page.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+
+import Loading from "./loading";
+import { ClientSideSuspense, RoomProvider } from "@liveblocks/react/suspense";
+import { useSearchParams } from "next/navigation";
+import Editor from "./tiptap/editor";
+import { Providers } from "./providers";
+import { useExampleUserId } from "./use-example-user-id";
+
+// Learn how to structure your collaborative Next.js app
+// https://liveblocks.io/docs/guides/how-to-use-liveblocks-with-nextjs-app-directory
+
+export default function Page() {
+  const userId = useExampleUserId();
+  const roomId = useExampleRoomId(
+    "liveblocks:channel-notifications-settings-examples:nextjs"
```

Comment:
`channel-notifications-settings-examples` needs to be updated here and I also saw it in other places so a quick find and replace should cover everything